src/Components/MainApp/App.js

```diff
@@ -7,42 +7,35 @@ import ExperimentTwo from '../ExperimentTwo/ExperimentTwo.js';
 
 function App() {
   // Button States
-  const [headerState, setHeaderState]=useState(true);
-  const [experimentOneButtonState,setExperimentOneButtonState ]=useState(false);
-  const [experimentTwoButtonState,setExperimentTwoButtonState]=useState(false);
+  const[experimentButtonState,setExperimentButtonState]=useState('0')
+  const experimentButtons=[
+    {
+      name: 'Experiment 1',
+      id : '1'
+    },
+    {
+      name: 'Experiment 2',
+      id:'2'
+    }
+  ];
 
-  const handleExperimentOneEvent=()=>
-  {
-    setExperimentOneButtonState(true);
-    setExperimentTwoButtonState(false);
-    setHeaderState(false);
+  const handleExperimentButtonState = (experimentButtonSelected)=>{
```

Comment:
Not sure if you really need this function - could just call the setState function directly

Reply:
Good shout. I agree and will therefore remove it